Sources/EpoxyLayoutGroups/Types/LayoutGroupUpdateAnimation.swift

```diff
@@ -0,0 +1,66 @@
+// Created by lindsay_pond on 5/13/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import UIKit
+
+public typealias GroupAnimationClosure = (_ animations: @escaping () -> Void, _ completion: @escaping (_ completed: Bool) -> Void) -> Void
+
+/// Provides parameter values, which can be passed to customize the animation on an Group's animated update (`_setItems(::)`).
```

Comment:
Since the `_setItems` method is private let's only document the public interface
```suggestion
/// Provides parameter values, which can be passed to the initializer of a Group
```